linux-anvil2/entrypoint

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -il
+
+# Create conda user with the same uid as the host, so the container can write
+# to mounted volumes
+# Adapted from https://denibertovic.com/posts/handling-permissions-with-docker-volumes/
+USER_ID=${HOST_USER_ID:-9001}
+useradd --shell /bin/bash -u $USER_ID -o -c "" -m conda
+export HOME=/home/conda
+export USER=conda
+export LOGNAME=conda
+export MAIL=/var/spool/mail/conda
+export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/conda/bin
+if [ "$(uname -m)" = "x86_64" ]; then
+   export supkg="gosu"
+else
+   export supkg="su-exec"
+fi
+chown conda:conda $HOME
+chown -R conda:conda /opt/conda
```

Comment:
This was changed in the linux-anvil image recently. https://github.com/conda-forge/docker-images/commit/2ba7672fff1f42139d62701e70a227f5d8feabd2